packages/modules/imodel-browser/src/types.ts

```diff
@@ -79,3 +95,18 @@ export type IModelSortOptionsKeys = "name";
 
 /** Object/function that configure IModel sorting behavior. */
 export type IModelSortOptions = SortOptions<IModelFull, IModelSortOptionsKeys>;
+
+export type ITwinType =
```

Comment:
Same as the other -- these aren't Types, they're subClasses. We should use the proper terminology

Reply:
Done